pyproject.toml

```diff
@@ -57,21 +57,23 @@ hypothesis = "^6.46.5"
 ipython = "^8.3.0"
 isort = "^5.10.1"
 jupyter = "^1.0.0"
-Markdown = "^3.3.7"
-markdown-include = "^0.6.0"
+Markdown = "==3.3.7"
```

Comment:
docs dependencies have been pinned to exact versions since, as it often happened in the past, the "docs stack" is fragile wrt version updates.